pyaedt/modules/Material.py

```diff
@@ -675,6 +685,38 @@ def __init__(self, parent, name, props=None):
                                                             MatProperties.get_defaultvalue(aedtname=property), None)
         pass
 
+    @property
+    def material_appearance(self):
+        """Material Appearance specified as an RGB list.
+
+        Returns
+        -------
+        type
+            Appearance of the material.
+        """
+        return self._material_appearance
+
+    @material_appearance.setter
+    def material_appearance(self, value):
+        if type(value) is list and len(value) == 3:
+            value_int = []
+            for i in value:
+                try:
+                    tmp = int(i)
+                except ValueError:
+                    raise
```

Comment:
We gain nothing here with a `try`/`except` by re raising the error.  We can either provide a helpful error:
```suggestion
                try:
                    tmp = int(i)
                except ValueError:
                    raise ValueError('List of RGB must be able to be converted to int')
```
Or simply:
tmp = int(i)

Also, `i` is not a good variable name.  Perhaps `rgb_item` works better.  Also, use `rgb` instead of `value` in the function signature.

Reply:
Done. Chosen the simple way. Renamed also `tmp` to `rgb_int`.